packages/medusa/src/api-v2/admin/customers/validators.ts

```diff
@@ -121,6 +121,11 @@ export class AdminPostCustomersReq {
   @IsString()
   @IsOptional()
   email?: string
```

Comment:
```suggestion
  email: string
```

Should we change this here as well?

Reply:
I don't think so since this is an optional field in the validator right?